xl_auth/templates/permissions/home.html

```diff
@@ -28,6 +29,7 @@
                     <td>{{ item.collection.code }}</td>
                     <td>{{ item.register }}</td>
                     <td>{{ item.catalogue }}</td>
+                    <td>{{ item.catalogue_admin }}</td>
```

Comment:
@mxtthias Thought a bit more about this term...
> user X with permission to **catalogue/katalogisera** in collection A
> user Y with permission to **register/beståndsregistrera** in collection A
> user Z with permission to **delegate/delegera** [to other users] in collectionA

With this name, it would read *"user Z with permission to **catalogue_admin/'katalogadministrera'** in collection A"*. Or *"**katalogisera-administrera** in collection A"*. Seems like a source incoherence and confusion with respect to long-term maintainability.

If "delegate" is too unspecific, what would you think about "cataloguing_mgnr", "cataloguing_admin", "collection_mgnr", or "collection_admin"?

As fallback options, I especially like "collection_mgnr", as it differentiates from "admin" and indicates clearly that *user Z with permission to **manage some stuff** related to collection A* (not tied specifically to either beståndsregistrering or katalogisering). In the future, if we decide to let them edit select other aspects of collection (apart from user privileges on it), it would be a trivial change (yet semantically coherent). E.g. editing properties such as friendly-name.

That's all from me, "[foolish consistency is the hobgoblin of little minds](https://www.python.org/dev/peps/pep-0008/#a-foolish-consistency-is-the-hobgoblin-of-little-minds)" and all. :) Merge-or-adjust as desirable.